src/Components/Users/UserAdd.tsx

```diff
@@ -124,6 +123,7 @@ export const UserAdd = (props: UserProps) => {
   const [isStateLoading, setIsStateLoading] = useState(false);
   const [isDistrictLoading, setIsDistrictLoading] = useState(false);
   const [isLocalbodyLoading, setIsLocalbodyLoading] = useState(false);
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Is it necessary to disable eslint for the next line? Can we instead fix the issue?  If the   variable is not required we can remove it.

Reply:
Have made the required changes in this PR https://github.com/coronasafe/care_fe/pull/3467
Created a new PR since there were a lot of code changes merged in develop branch, and due to that there were too many merge conflicts with this branch